content/geometry/introduction.adoc

```diff
@@ -9,17 +9,17 @@ endif::[]
 
 == Motivation
 
-A _3D asset_ is a digital entity that includes a 3D geometric representation of an object (also known as _3D model_ or **3D Data File**) along with its attributes and metadata. 
+A _3D asset_ is a digital entity that includes a 3D geometric representation of an object (also known as _3D model_ or 3D Data File) along with its attributes and metadata. 
 Common formats for storing the geometry and visual materials of _3D models_ include glTF, FBX, and USD.
 
 However, these formats do not provide a standardized way to structure the geometry or provide extensive metadata storage and exchange capabilities. 
 The {THIS_STANDARD} standard addresses these needs.
 
 This chapter includes all geometry related definitions of {THIS_STANDARD}, marked in red in <<fig-openmaterial-geometry-overview>>.
 
-The {THIS_STANDARD} **Asset File** (for example, "omg_my-model.xoma") contains essential information relevant to the display and interactivity attributes of the _3D model_ in various computer applications.
+The {THIS_STANDARD} asset file, for example, "omg_my-model.xoma", contains essential information relevant to the display and interactivity attributes of the _3D model_ in various computer applications.
```

Comment:
The "omg_" prefix needs to be removed, we dropped that concept.

Reply:
Changed accordingly.